assets/palm2.yaml

```diff
@@ -0,0 +1,30 @@
+---
+
+- type: model
+  name: PaLM 2
+  organization: Google
+  description: PaLM2 is a new state-of-the-art language model that has better multilingual
+    and reasoning capabilities and is more compute-efficient than its predecessor
+    PaLM. PaLM 2 is a Transformer-based model trained using a mixture of objectives
+    similar to UL2.
+  created_date: 2023-05-10
+  url: https://blog.google/technology/ai/google-palm-2-ai-large-language-model/
+  model_card: https://ai.google/static/documents/palm2techreport.pdf
+  modality: text (input and output)
+  analysis: Reports results on standard code benchmarks across a variety of programming
+    languages.
+  size: 3.6T parameters
+  dependencies: [PaLM 2 Dataset]
```

Comment:
Dataset can be lower-cased. 

Add a node for PaLM 2 dataset as well and document

Reply:
I couldn't find sufficient information about the PaLM 2 dataset, only that it's trained on a diverse set of sources (Section 3 of https://ai.google/static/documents/palm2techreport.pdf).